dcs/task.py

```diff
@@ -1077,6 +1082,39 @@ def __eq__(self, other) -> bool:
         return False
 
 
+class CarpetBombing(Task):
+    Id = "CarpetBombing"
+
+    class AttackType(Enum):
+        Carpet = "Carpet",
+
+        def __eq__(self, other: Any) -> bool:
+            if isinstance(other, str):
+                return self.value == other
+            return self == other
+
+    def __init__(self, altitude: int = 2000, altitude_enabled: bool = False,
+                 attack_qty: int = 1, attack_qty_limit: bool = False,
```

Comment:
This doesn't appear to be an option in the ME. What is this?

Reply:
If you can't explain what it does, don't put it in the API. The implementation can fill this in, but I've got no idea what this does (and it sounds like you don't either), so it's not a useful parameter.

I'd recommend creating a mission using this task, look at the lua, and hard code whatever it uses. Make a few of them with different settings for the weapon type, probably, and make sure they're all the same.

If the value is not constant and you can't figure out a pattern, I've got no idea what to do here, but punting that question to the caller isn't going to work.